README.md

```diff
@@ -97,6 +97,9 @@ The following prerequisites are required to build and deploy the plugins from so
 - **llvm**
   - Compiling the bpf progs for Kind.
   - Install on Ubuntu: `apt install llvm`
+- **libxdp**
+  - Same as Libbpf, to load and uload the XDP program onto the network device.
```

Comment:
*unload (spellcheck)

 Might be worth adding just a brief about what we're doing - installing libbpf, the wget packages and libxdp.

Also, I think we should be doing a libbpf check on the host first as we're including it as the first step of the libxdp installation:

If not present = first install libbpf 
else =  install libxdp

If we're to break the installation process to 2 sections (libbpf and libxdp) both steps will be required for the ci-public.yml. 

@garyloug, we might discuss this further.